packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -257,7 +257,7 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 	};
 
 	const displaySettings = useDisplaySettings(breakpoints) || {};
-	const theme = deepmerge(themeOverride, deepmerge(props?.theme || {}, displaySettings?.theme || {}));
```

Comment:
looks like themeOverride is no longer used, I think this is still needed right? Maybe we need to do the same arrayMerge twice here? not sure what theming example you were using to test this all with, but that seems to pass tests and everything fine. 

`
const theme =  deepmerge(themeOverride, deepmerge(props?.theme || {}, displaySettings?.theme || {}, { arrayMerge: (destinationArray, sourceArray) => sourceArray }), { arrayMerge: (destinationArray, sourceArray) => sourceArray });
`

Reply:
Oh good catch!

Reply:
Corrected!